src/functional.js

```diff
@@ -81,3 +82,10 @@ export const timeit = (handler, f) => async (...args) => {
   handler(getTimestampMilliseconds() - started);
   return result;
 };
+
+export const asyncTap = (f) => async (x) => {
+  await f(x);
```

Comment:
must we wait here?

Reply:
That's the whole idea. To do a side effect while waiting. If you don't want to wait use regular tap.